modules/vfs-class-loader/src/test/java/org/apache/accumulo/classloader/vfs/context/ReloadingVFSContextClassLoaderFactoryTest.java

```diff
@@ -64,43 +62,37 @@ public TestReloadingVFSContextClassLoaderFactory(String dir) {
       this.dir = dir;
     }
 
+    @SuppressFBWarnings(value = "DP_CREATE_CLASSLOADER_INSIDE_DO_PRIVILEGED",
+        justification = "Security Manager is deprecated for removal as of JDK 17")
```

Comment:
I wonder if updating spotbugs would obviate the need for this warning.

Reply:
I tried that and it did not. There seems to still be open issue for it: https://github.com/spotbugs/spotbugs/issues/1515

Furthermore it caused SpotBugs to throw a bunch more warnings so I figured we'd update it separately.

Reply:
I could also add an exclude filter file instead of an annotation to cover it everywhere, not sure which is better.